pkg/messages/messages.go

```diff
@@ -42,9 +44,23 @@ func (msg Message) Error() string {
 
 // Info ...
 func (msg Message) Info() string {
+	/*If the BackendError is from library e.g BackendError: {Trace Code:920df6e8-6be9-4b4a-89e4-837ecb3f513d,
+	Code:volume_profile_capacity_iops_invalid,Description:The capacity or IOPS specified in the request is not valid for the custom profile,
```

Comment:
can we remove reference of any storage related comments ?
apart from this PR looks good

Reply:
Updated @arahamad